stubs/unidiff/unidiff/constants.pyi

```diff
@@ -0,0 +1,22 @@
+from re import Pattern
+
+RE_SOURCE_FILENAME: Pattern[str]
+RE_TARGET_FILENAME: Pattern[str]
+RE_DIFF_GIT_HEADER: Pattern[str]
+RE_DIFF_GIT_HEADER_URI_LIKE: Pattern[str]
+RE_DIFF_GIT_HEADER_NO_PREFIX: Pattern[str]
+RE_DIFF_GIT_DELETED_FILE: Pattern[str]
+RE_DIFF_GIT_NEW_FILE: Pattern[str]
+RE_HUNK_HEADER: Pattern[str]
+RE_HUNK_BODY_LINE: Pattern[str]
+RE_HUNK_EMPTY_BODY_LINE: Pattern[str]
+RE_NO_NEWLINE_MARKER: Pattern[str]
+RE_BINARY_DIFF: Pattern[str]
+DEFAULT_ENCODING: str
+DEV_NULL: str
+LINE_TYPE_ADDED: str
+LINE_TYPE_REMOVED: str
+LINE_TYPE_CONTEXT: str
+LINE_TYPE_EMPTY: str
+LINE_TYPE_NO_NEWLINE: str
+LINE_VALUE_NO_NEWLINE: str
```

Comment:
These are all constants, perhaps mark them as such with `Final`?

Reply:
Done, thanks for the review!